react/features/authentication/components/styles.js

```diff
@@ -26,25 +10,24 @@ export default createStyleSheet({
      * feature authentication.
      */
     dialogText: {
-        ...text,
         margin: BoxModel.margin,
         marginTop: BoxModel.margin * 2
     },
 
     /**
-     * The style of {@code LoginDialog}.
+     * The style used when an error message is rendered.
      */
-    loginDialog: {
-        ...dialog,
-        flex: 0,
-        flexDirection: 'column'
+    errorMessage: {
+        color: ColorPalette.red
```

Comment:
Don't you wanna brand this color?

Reply:
I'm not sure where to put it TBH.

Reply:
i'd put it in the top level definition with a name something like '`errorMessageText` or similar